ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/emailinput/v1/emailinput/emailinput.html

```diff
@@ -21,18 +21,20 @@
      data-sly-use.shortDescription="${renderer.shortDescriptionPath}"
      data-sly-use.longDescription="${renderer.longDescriptionPath}"
      data-sly-use.errorMessage="${renderer.errorMessagePath}"
-     data-sly-use.questionMark="${renderer.questionMarkPath}"></sly>
+     data-sly-use.questionMark="${renderer.questionMarkPath}"
+     data-sly-test.widgetId="${'{0}-{1}' @ format=[email.id, 'widget']}"></sly>
```

Comment:
if we are putting in if condition to determine rendering, we must add testcases for the scenarios to ensure no future regressions

Reply:
@deepprakash345 
There are known test case failures. I have changed the value of the “for” attribute for all components. Once the PR is approved, I will update the test cases and add new test cases.